src/pro/sky/java/course1/Main.java

```diff
@@ -3,6 +3,132 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        //work 1
+
+        int clientOS = 1;
+        if(clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        } else
+            if(clientOS == 0){
```

Comment:
лучше объединить в одну строку цепочку условного оператора, чтобы избежать неявной вложенности
```suggestion
        } else if(clientOS == 0){
```
Рекомендую использовать ctrl + alt + L для автоформатирования кода